Glimpse.ApplicationInsights.Sample/packages.config

```diff
@@ -13,6 +13,7 @@
   <package id="Glimpse.AspNet" version="1.9.2" targetFramework="net45" />
   <package id="Glimpse.EF6" version="1.6.5" targetFramework="net45" />
   <package id="Glimpse.Mvc5" version="1.5.3" targetFramework="net45" />
+  <package id="Glimpse_ApplicationInsights_Extension" version="1.0.1" targetFramework="net45" />
```

Comment:
let's change name to Glimpse.ApplicationInsights.Extension to align with the naming convention